### PR TITLE
Fix SvelteKit Upgrade

### PR DESCRIPTION
**Pull request by Winterhart**
- Add peer dependencies
- Force to install some sub dependencies at specific version

### How to test?

```bash
git clone https://github.com/evidence-dev/template  
git checkout fix-sveltekit-upgrade 
npm install
npm run dev
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,16 @@
   },
   "type": "module",
   "dependencies": {
-    "@evidence-dev/evidence": "^12.0.1"
+    "@evidence-dev/components": "latest",
```
Comment (Winterhart) [inline package.json:16]: Comes from: https://github.com/evidence-dev/evidence/blob/main/packages/evidence/package.json#L43

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "build": "evidence build",
     "build:strict": "evidence build:strict",
-    "dev": "evidence dev -o",
```
Comment (archiewood) [inline package.json:7]: any reason you're removing this flag?

Comment (Winterhart) [reply package.json:7]: It's not available anymore in the `vite dev` command I can put:
```
evidence dev --open /
```

which would replace `-o`

Comment (hughess) [reply package.json:7]: That would work - it's a good thing to have for first-time users

**Comment by archiewood** [comment]
@Winterhart works for me. 

Out of interest - why is the addition of the extra deps?

File: package.json (hunk 2 above)
Comment (archiewood) [reply package.json:7]: added

**Review by archiewood — APPROVED** [review]

**Comment by archiewood** [comment]
@gregwdata added your DB file here and seems to be working! 🙏🏼

image

**Comment by Winterhart** [comment]
> @Winterhart works for me. 
> 
> 
> 
> Out of interest - why is the addition of the extra deps?

I am overwriting the dependencies to make sure we have the latest that include the fix.

We will rework on those dependencies in the upcoming cycles with the restructuring story.

**Pull request merged**